### PR TITLE
Add service selection

### LINKED ISSUE #39: Implement create Linode flow plan selection

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #39

### CODE REVIEW

File: scss/components/distro.scss [hunk 1]
```diff
@@ -52,20 +70,15 @@
...
-}
-
-.datacenters {
-    padding: $main-padding-tb $main-padding-lr;
+.plan {
+    text-align: center;
 
-    h3 {
-        margin-left: 1rem;
+    & > div {
```
Comment (SirCmpwn) [inline scss/components/distro.scss]: I still want you to put a class on this div. Put a class on this div.

Comment (eatonphil) [inline scss/components/distro.scss]: Whoops, must have forgotten a rebase.

Comment (eatonphil) [inline scss/components/distro.scss]: This is a different div than the header label. This is the contents of the plan card.

**Comment by eatonphil** [comment]
screen shot 2016-06-30 at 3 01 04 pm

File: scss/components/distro.scss (hunk 1 above)
Comment (SirCmpwn) [inline scss/components/distro.scss]: I see. Well, let's slap a class on both of them.

**Comment by eatonphil** [comment]
ready for review/merge

File: src/linodes/components/ServiceSelection.js [hunk 2]
```diff
@@ -15,14 +14,58 @@ export default class ServiceSelection extends Component {
...
+    const {
+      label,
+      transfer,
+      disk,
+      monthly_price: monthlyPrice,
+      hourly_price: hourlyPrice,
+      ram,
+      vcpus,
+      id,
+    } = plan;
```
Comment (SirCmpwn) [inline src/linodes/components/ServiceSelection.js]: At this point I'd just use plan.label et al. We don't have to deconstruct everything.

File: test/linodes/components/ServiceSelection.spec.js [hunk 3]
```diff
@@ -1,11 +1,68 @@
...
+      id: 'service_114',
+      label: 'Linode 2048',
+      mbits_out: 25,
+      monthly_price: 1000,
+      ram: 2048, // larger plan
+      service_type: 'linode',
+      transfer: 2000,
+      vcpus: 1,
+    },
+  };
```
Comment (SirCmpwn) [inline test/linodes/components/ServiceSelection.spec.js]: \>\>\>test/data.js

File: test/linodes/layouts/CreateLinodePage.spec.js [hunk 4]
```diff
@@ -37,6 +42,22 @@ describe('linodes/layout/CreateLinodePage', () => {
...
+          label: 'Linode 1024',
+          mbits_out: 25,
+          monthly_price: 1000,
+          ram: 1024,
+          service_type: 'linode',
+          transfer: 2000,
+          vcpus: 1,
+        },
+      },
+    },
```
Comment (SirCmpwn) [inline test/linodes/layouts/CreateLinodePage.spec.js]: :boom: :exclamation: :exclamation: whoop whoop DRY alarm whoop whoop :exclamation: :exclamation: :boom:

Comment (eatonphil) [inline test/linodes/layouts/CreateLinodePage.spec.js]: I deserved that

**Pull request merged**